de.metas.adempiere.adempiere/base/src/test/java/de/metas/location/geocoding/provider/openstreetmap/NominatimOSMGeocodingProviderImplTest.java

```diff
@@ -114,7 +110,6 @@ void bestCoordinatesFindsMetasDEByFullAddress()
 				.contains(expectedCoordinates);
 	}
 
-
 	@Test
 	@DisplayName("bestCoordinates cannot find metasRO by postal and wrong country")
```

Comment:
nice, i wasn't aware of that annotation

Reply:
only available in junit5, but it's nice coz the IDE shows the "display name" instead of the test name.